CHANGES.md

```diff
@@ -1,3 +1,9 @@
+Unreleased
+==========
+
+Note that this release includes a change in Synapse to use Redis as a cache, as well as a pub/sub mechanism, if Redis support is enabled. No action is needed by server administrators.
```

Comment:
I don't know if its worth saying that we don't expect much increase in memory usage? Matrix.org is currently running at ~20MB.

Reply:
It might be worth mentioning that we don't expect the Redis usage to dramatically increase.

Reply:
Done